backend/cms/templates/pages/page.html

```diff
@@ -316,8 +319,10 @@ <h2 class="heading font-normal">
                         editor.selection.setContent('<span class="notranslate" translate="no">' + editor.selection.getContent() + '</span>');
                     }
                 }
-            }),
```

Comment:
@timoludwig shouldn't this be a ; ?

Reply:
Yes, this is a syntax error. Probably not changing anything, but better fix it :wink: